docs/src/features.md

```diff
@@ -182,11 +182,56 @@ Add these settings to LSP settings, your Sublime settings, Syntax-specific setti
 * `highlight_active_signature_parameter`: *highlight the active parameter of the currently active signature*
 * `document_highlight_style`: *document highlight style: "box", "underline", "stippled", "squiggly" or ""*
-* `document_highlight_scopes`: *customize your sublime text scopes for document highlighting*
 * `diagnostics_gutter_marker` `"dot"` *gutter marker for code diagnostics: "dot", "circle", "bookmark", "sign" or ""*
 * `show_symbol_action_links` `false` *show links to symbol actions like go to, references and rename in the hover popup*
 * `disabled_capabilities`, `[]` *Turn off client capabilities (features): "hover", "completion", "documentHighlight", "colorProvider", "signatureHelp", "codeLensProvider", "codeActionProvider"*
 * `log_debug` `false` *show debug logging in the sublime console*
 * `log_server` `[]` *log communication from and to language servers*
 * `log_stderr` `false` *show language server stderr output in the console*
 * `log_max_size` `8192` *max  number of characters of payloads to print*
+
+### Color configurations
+
+Some features use TextMate scopes to control the colors (underline, background or text color) of styled regions in a document or popup.
+Colors can be customized by adding a rule for these scopes into your color scheme, see https://www.sublimetext.com/docs/3/color_schemes.html#customization.
+
+The following tables give an overview about the scope names used by LSP.
+
+#### Document Highlights
+
+!!! info "This feature is only available if the server has the *documentHighlightProvider* capability."
+    Highlights other occurrences of the symbol at a given cursor position.
+
+| scope | DocumentHighlightKind | description |
+| ----- | --------------------- | ----------- |
+| `markup.highlight.text.lsp` | Text | A textual occurrence |
+| `markup.highlight.read.lsp` | Read | Read-access of a symbol, like reading a variable |
+| `markup.highlight.write.lsp` | Write | Write-access of a symbol, like writing to a variable |
+
+!!! note
+    If `document_highlight_style` is set to "fill" in the LSP settings, the highlighting color can be controlled via the "background" color from a color scheme rule for the listed scopes.
+
+#### Diagnostics
+
+| scope | DiagnosticSeverity | description |
+| ----- | ------------------ | ----------- |
+| `markup.error.lsp` | Error | Reports an error |
+| `markup.warning.lsp` | Warning | Reports a warning |
+| `markup.info.lsp` | Information | Reports an information |
+| `markup.info.hint.lsp` | Hint | Reports a hint |
+
+!!! note
+    If `diagnostics_highlight_style` is set to "fill" in the LSP settings, the highlighting color can be controlled via the "background" color from a color scheme rule for the listed scopes.
+
+#### Signature Help
+
+| scope | description |
+| ----- | ----------- |
+| `entity.name.function.sighelp.lsp` | Function name in the signature help popup |
```

Comment:
Maybe not use abbreviation like "sighelp"?

Reply:
This was added by @rwols in #1576. I do find it useful, so that it can be easily targeted by color schemes. Or do you mean write it out as e.g. "entity.name.function.signature-help.lsp" instead of "sighelp"?

But besides that, looking at the docs now, I recommend to change "markup.codelens.accent" to "markup.accent.codelens.lsp". The "markup.accent" looks general enough so it could be reused by other features than code lens, or other plugins too, and the "lsp" suffix to keep all scopes from LSP consistent.

Reply:
> Or do you mean write it out as e.g. "entity.name.function.signature-help.lsp" instead of "sighelp"?

Yeah I meant that, rename "sighelp" to "signature-help". (abbreviations are sometimes hard to understand) 
But because this was already added I don't want to introduce breaking changes, so ignore this comment.

> <small>because this was recently added 11 feb. maybe worth reconsidering renaming... but again this is not a big deal.</small>